### PR TITLE
Deprecate the unused endpoint

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
The endpoint is useless and does nothing. See https://github.com/appium/appium-uiautomator2-driver/pull/222 for more details:

### CODE REVIEW

File: app/src/main/java/io/appium/uiautomator2/handler/AppStrings.java [hunk 1]
```diff
@@ -1,51 +1,22 @@
...
-            dataInput.close();
-
-            JSONObject appStrings = new JSONObject(new String(jsonBytes, "UTF-8"));
-            Logger.debug("json loading complete ");
-            return new AppiumResponse(getSessionId(request), WDStatus.SUCCESS, appStrings);
-        } catch (IOException e) {
-            Logger.error("Error loading json from " + filePath + " : " + e);
-            throw new UiAutomator2Exception(e);
-        }
+        return new AppiumResponse(getSessionId(request), WDStatus.SUCCESS, Collections.emptyMap());
```
Comment (dpgraham) [inline app/src/main/java/io/appium/uiautomator2/handler/AppStrings.java]: If a user tries calling this endpoint, will uiautomator2-driver provide a deprecation notice?

Comment (mykola-mokhnach) [reply app/src/main/java/io/appium/uiautomator2/handler/AppStrings.java]: What would be the best to do? Write a string to logcat?

Comment (dpgraham) [reply app/src/main/java/io/appium/uiautomator2/handler/AppStrings.java]: Maybe a warning, `Logger.warn("The unlock endpoint is does nothing and will be removed in a future release")`.... just so the user knows it's not doing anything.

Comment (mykola-mokhnach) [reply app/src/main/java/io/appium/uiautomator2/handler/AppStrings.java]: added

Comment (dpgraham) [reply app/src/main/java/io/appium/uiautomator2/handler/AppStrings.java]: Oh, it's /strings... I thought it was unlock.... same thing I guess.

**Comment by mykola-mokhnach** [comment]
Would you prefer to remove it completely and raise major module version instead

**Comment by mykola-mokhnach** [comment]
@jlipps the handler has been completely deleted

**Pull request merged**